dashboard/callbacks.py

```diff
@@ -176,11 +176,9 @@ def upload_label(filename):
 # these computations are cached in a globally available
 # redis memory store which is available across processes
 # and for all time.
-# pylint: disable=dangerous-default-value
-# pylint: disable=too-many-arguments
 @cache.memoize()
 def global_store_i(method_sel, model_path, image_test,
-                   labels=list(range(2)), axis_labels={2: 'channels'},
+                   labels=(0, 1), axis_labels={2: 'channels'},
```

Comment:
You can make it `[0, 1]` so that you needn't do the `labels = list(labels)` call below.

Reply:
It's dangerous to do that, and not recommended. A list is mutable and `labels` only gets initialized during import time. So if any function downstream updates the labels (i.e. `labels[0]='foo'`), your default arguments are now `labels=['foo', 1].

Reply:
Ah great point, hadn't thought about that!